mappings/InventoryOwner.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS net/minecraft/class_6067 InventoryOwner
```

Comment:
think enigma glitched and lost the package name

Reply:
damnit i hate how enigma selects the whole dang thing now
plus whenever you start typing something it selects the first letter you type so you always end up overwriting it
it's so frustrating

Reply:
Yeah, enigma's typing in place to rename is widely criticized